bin/oref0-autotune-prep.js

```diff
@@ -87,6 +87,15 @@ if (!module.parent) {
         }
     }
 
+    // get insulin curve from pump profile that is maintained
+    profile_data.curve = pumpprofile_data.curve;
+
+    // if not tuning the insulinEndTime and insulinPeakTime, then use data from pump profile
+    if (!tune_insulin_curve) {
```

Comment:
This would change those values back to defaults on subsequent runs if you tune insulin curve manually and have autotune running on cron. Do we want that? Or should we stick with the tuned insulin curve?

Reply:
I suspect the more common use case is that someone will only have DIA set in the pump, and then will run tune_insulin_curve in a manual autotune run.  I don't want to impose too many hoops: it seems like the default behavior should be "if you've tuned insulinPeakTime or insulinEndTime, keep using the tuned values"...

Reply:
That makes sense for that to be the more common use case.

Any thoughts on how to make the fact that curve values have been tuned persist so it could test for that case?

Reply:
Would it be as simple as checking whether useCustomPeakTime is set (whatever it's called exactly) instead of checking to see if we're doing tune_insulin_curve at the moment?

Reply:
Can I add a flag to the autotune output profile it would set to true if tune_insulin_curve had been run? Testing the flag instead of the argument would then be used to determine if it needs to keep the dia and peak from the previous autotune profile vs. pull it in from the pumpprofile.

Reply:
Yep, I was overthinking it. Changed to use useCustomPeakTime.